packages/api/tests/models/test-pdrs-model.js

```diff
@@ -250,6 +250,46 @@ test(
   }
 );
 
+test(
+  'storePdrFromCumulusMessage does not update if PDR record is from an older, prior execution',
+  async (t) => {
+    const pdrName = randomId('pdr');
+    const stateMachine = randomId('parsePdr');
+    const execution = randomId('exec');
+
+    const initialMsg = createPdrMessage({
+      execution,
+      stateMachine,
+      numCompletedExecutions: 3,
+      status: 'completed',
+      createdAtTime: (Date.now() + 10000000000),
+    });
+
+    initialMsg.payload.pdr = {
+      name: pdrName,
+    };
+
+    await pdrsModel.storePdrFromCumulusMessage(initialMsg);
+
+    const newMsg = createPdrMessage({
+      execution: randomId('newExec'),
+      stateMachine,
+      numRunningExecutions: 2,
+      status: 'running',
```

Comment:
I think we should pass in the same `Date.now()` as `createdAtTime` to this factory invocation to be sure that it is an older time

Reply:
Fair point, this is reliant on the helper default for logic that's being explicitly testing.

f251723ef